docs/src/index.md

```diff
@@ -0,0 +1,17 @@
+# Miden Faucet
+
+The Miden faucet contains a library and a binary CLI.
```

Comment:
I would expand this a bit to explain what Miden faucet is, what it does, and why it is needed. Probably doesn't need to be more than 2 - 3 paragraphs.